R/plate-utils.R

```diff
@@ -77,12 +77,32 @@ init_meta <- function(plate) {
   if (is.null(plate_meta)) {
     plate_meta <- DEFAULT_PLATE_META
   } else {
-    meta_cols_keep <- c("well", "sample")
+    meta_cols_keep <- c("well", "sample", "target_ch1", "target_ch2")
     tryCatch({
+      
+      if (colnames(plate_meta)[5] == 'typeassay' | 
```

Comment:
Does the number 5 have to be hardcoded or would it be ok to just see if this variable exists at all in any of the columns?